docs/tutorials.rst

```diff
@@ -0,0 +1,55 @@
+Tutorials
+==============
+
+The CAP form
+----------------
+There are four main section to document a physics analysis on the CAP form, they are basic information, data provenance, analysis software and documentation. However, each form is tailored to the needs of its experiment. In the following examples we use form snippets from different experiments to demonstrate their flexibility and use.
+
+Basic information
+~~~~~~~~~~~~~~~~~~~
+
+Basic information captures name, measurement, proponents, the status of the application and other.
+
+.. image:: _static/fig1.png
+
+Data provenance
+~~~~~~~~~~~~~~~~~~~
+
+The section on data provenance captures which data sets are used in the analysis and how. By clicking the button **+** a large number of data sets can be documented.
+
+.. image:: _static/fig2.png
+
+The CAP form implements the links to the existing experimental databases. In the example below an analyst can import their analysis information from the CADI database at CMS.
+
+.. image:: _static/fig3.png
+
+Analysis software
+~~~~~~~~~~~~~~~~~~~
+
+Analysis software can be captured directly from git repositories. Special attention has been taken to ensure analysis automation and creating analysis workflow which can also be captured using the form.
```

Comment:
After "analysis automation" the syntax kind of confuses me. Could you perhaps try to reword this?

Reply:
😂 you are right